docs/MIGRATION-v0.8.0.md

```
@@ -0,0 +1,379 @@
+{
+  "mcpServers": {
+    "ynab": {
+      "command": "npx",
+      "args": ["ynab-mcp-server"],
+      "env": {
+        "YNAB_ACCESS_TOKEN": "your-token-here",
+
+        // New v0.8.0 cache options (all optional)
+        "YNAB_MCP_CACHE_DEFAULT_TTL_MS": "300000",     // 5 minutes (default)
+        "YNAB_MCP_CACHE_MAX_ENTRIES": "1000",          // 1000 entries (default)
+        "YNAB_MCP_CACHE_STALE_MS": "120000"             // 2 minutes (default)
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)